src/components/Homepage/CategoryGrid.vue

```diff
@@ -77,7 +80,7 @@ import getCacheKey from '@/util/getCacheKey';
 
 export default {
 	name: 'CategoryGrid',
-	serverCacheKey: () => getCacheKey('CategoryGrid'),
+	serverCacheKey: () => getCacheKey('CategoryGrid_2020-03-10'),
```

Comment:
Is there a convention for changing the serverCacheKey?

Reply:
You don't need to do this because you're changing the component entirely, not setting something up where it changes from props. The getCacheKey method appends the current git commit hash to the cache key, so you don't need to ever worry about previous versions of the component being used.

Reply:
@emuvente cool, thank you!